DentOS_Framework/DentOsTestbedLib/src/dent_os_testbed/lib/traffic/ixnetwork/ixnetwork_ixia_client_impl.py

```diff
@@ -100,6 +101,18 @@ def run_connect(self, device, command, *argv, **kwarg):
             device.applog.info("Assigning ports")
             IxnetworkIxiaClientImpl.ixnet.AssignPorts(pports, [], vport_hrefs, True)
             for port, vport in vports.items():
+                # The port numbers for mixed mode are
+                # divided equally between copper and fiber
```

Comment:
If mixed each port media type must be specified

Reply:
done